lib/routes/account.js

```diff
@@ -538,6 +540,29 @@ module.exports = function (
           }
         }
 
+        function sendVerifyAccountEmail() {
+          // For legacy clients, behavior is to not send an email to verify their account
+          // and have the requestor send it.
+          var sendEmailIfUnverified = false
```

Comment:
Yea eventually we do. This has to be `false` until the content-server has been updated to send the `sendEmailIfUnverified` param. Once that is merged, I think it is safe to switch this to `true`.